src/Server/index.ts

```diff
@@ -192,19 +192,23 @@ export class Server implements ServerContract {
    * server
    */
   public async handle(req: IncomingMessage, res: ServerResponse): Promise<void> {
-    /*
-     * Reset accept header when `forceContentNegotiationToJSON = true`
-     */
-    if (this.httpConfig.forceContentNegotiationTo) {
-      req.headers['accept'] = this.httpConfig.forceContentNegotiationTo
-    }
-
     const request = new Request(req, res, this.encryption, this.httpConfig)
     const response = new Response(req, res, this.encryption, this.httpConfig, this.router)
 
     const requestAction = this.getProfilerRow(request)
     const ctx = this.getContext(request, response, requestAction)
 
+    /*
+     * Reset accept header when `forceContentNegotiationTo` is defined
+     */
+    if (this.httpConfig.forceContentNegotiationTo) {
```

Comment:
How about changing the block of code to this.

```ts
const accepts = this.httpConfig.forceContentNegotiationTo
if (accepts) {
  req.headers['accepts'] = typeof (accepts) === 'function' ? accepts(ctx) : accepts
}
```

- The variable is named `accepts` because it is not a wrong name. However, the config uses a verbose name as not every understand the `accepts` header intuitively. 
- We can still update `req.headers['accepts']`, as the `req` object is passed by reference to `ctx.request`

Reply:
Agreed, the change has been done. 👍🏻